js/app.js

```diff
@@ -1,70 +1,70 @@
 
 (function(module){
 
-projects = [];
+  projects = [];
 
-function ProjectItem(obj) {
-  this.title = obj.title;
-  this.author = obj.author;
-  this.projectUrl = obj.projectUrl;
-  this.description = obj.description;
-  this.publishedOn = obj.publishedOn;
-  this.boot = obj.boot;
-};
+  function ProjectItem(obj) {
+    this.title = obj.title;
+    this.author = obj.author;
+    this.projectUrl = obj.projectUrl;
+    this.description = obj.description;
+    this.publishedOn = obj.publishedOn;
+    this.boot = obj.boot;
+  };
 
 // removed old commented code
 
-ProjectItem.prototype.toHtml = function() {
-  var template = Handlebars.compile($('#article-template').text());
+  ProjectItem.prototype.toHtml = function() {
+    var template = Handlebars.compile($('#article-template').text());
 
-  this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
-  this.monthsAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/30/1000);
+    this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
+    this.monthsAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/30/1000);
 
-  this.complete = this.daysAgo + ' days ago / ' + this.monthsAgo + ' months ago';
+    this.complete = this.daysAgo + ' days ago / ' + this.monthsAgo + ' months ago';
 
     return template(this);
 
     // removed old code
 
-};
-
-ProjectItem.loadAll = function(data) {
-
-
-  projects = data.map(function(stuff) {
-    console.log(data);
-    console.log(stuff);
-    return new ProjectItem(stuff);
-  })
-  var boot = data.map(function(a){
-    return parseInt(a.boot);
-  })
-  var bootTotal = boot.reduce(function(a,b){
-    return a + b;
-  });
-  console.log(boot);
-  console.log(bootTotal);
-  $('.footer').append(' There are ' + bootTotal + ' boots total in this biznatch!! ');
+  };
+
+  ProjectItem.loadAll = function(data) {
+
+
+    projects = data.map(function(stuff) {
+      console.log(data);
+      console.log(stuff);
+      return new ProjectItem(stuff);
+    });
+    var boot = data.map(function(a){
+      return parseInt(a.boot);
+    });
+    var bootTotal = boot.reduce(function(a,b){
+      return a + b;
+    });
+    console.log(boot);
+    console.log(bootTotal);
+    $('.footer').append(' There are ' + bootTotal + ' boots total in this biznatch!! ');
   // data.forEach(function(ele) {
   //   projects.push(new ProjectItem(ele));
   // });
```

Comment:
im using part of it i think i cleaned it up last night

Date: Wed, 17 Feb 2016 11:20:14 -0800
From: notifications@github.com
To: 301portfolio@noreply.github.com
CC: dzknj@outlook.com
Subject: Re: [301portfolio] linted all (#21)

In js/app.js:

> +
> +
> +    projects = data.map(function(stuff) {
> +      console.log(data);
> +      console.log(stuff);
> +      return new ProjectItem(stuff);
> +    });
> +    var boot = data.map(function(a){
> +      return parseInt(a.boot);
> +    });
> +    var bootTotal = boot.reduce(function(a,b){
> +      return a + b;
> +    });
> +    console.log(boot);
> +    console.log(bootTotal);
> +    $('.footer').append(' There are ' + bootTotal + ' boots total in this biznatch!! ');
>    // data.forEach(function(ele) {
>    //   projects.push(new ProjectItem(ele));
>    // });

Are you still using this code block?


—
Reply to this email directly or view it on GitHub.